soc/intel/intel_adsp/ace/Kconfig.defconfig.ace20_lnl

```diff
@@ -6,4 +6,11 @@ if SOC_INTEL_ACE20_LNL
 config MP_MAX_NUM_CPUS
 	default 5
 
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 393216000 if XTENSA_TIMER
+	default 38400000 if INTEL_ADSP_TIMER
+
+config XTENSA_CCOUNT_HZ
```

Comment:
The patch is ok. The CCOUNT_HZ definition is a not ideal as it's only used in the arch_timing implementation (added in commit 12eda76939d5f8cc72ebacda3f834fae40832a08 ) and it's not really accurate when DSP is run at a lower clock.  Oh well, not really part of this PR.

Reply:
Yes, actually we dont need all Hz configs because we can read correct value from capabilities registers but all of this is not part of this PR :) Just fixing PTL values here

Reply:
> because we can read correct value from capabilities registers

Funny that, I was just wondering why this was in Kconfig instead of DeviceTree.
https://docs.zephyrproject.org/latest/build/dts/dt-vs-kconfig.html

Reply:
> Just fixing PTL values here

Actually: something would be quite useful: adding a "TODO: read these from capabilities registers" to save whoever will look at this again confusion and time.